temperature/crud.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+from sqlalchemy import select
+from sqlalchemy.orm import selectinload
+
+from city.crud import get_all_cities
```

Comment:
The import statement `from city.crud import get_all_cities` is correct if you are using this function to retrieve city data for temperature updates. Ensure that the `get_all_cities` function is implemented correctly in the `city.crud` module.